jenkins-agent.pkr.hcl

```diff
@@ -56,9 +56,9 @@ variable "azure_subscription_id" {
   default = ""
   type    = string
 }
-variable "azure_image_version" {
+variable "image_version" {
   type    = string
-  default = "0.0.2"
+  default = "0.0.1" # Default is a valid version to not fail azure validation
```

Comment:
This seems weird to me to replace 0.0.2 by 0.0.1

Reply:
Good point. The reason is that we started with the version "raw" in the code, and now the paradigm changed.
WDYT about setting the initial default value as `0.1.0` instead?